sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -761,6 +761,12 @@ object SQLConf {
       .checkValue(threshold => threshold >= 0, "The threshold must not be negative.")
       .createWithDefault(10)
 
+  val PARQUET_AGGREGATE_PUSHDOWN_ENABLED = buildConf("spark.sql.parquet.aggregatePushdown")
+    .doc("Enables Parquet aggregate push-down optimization when set to true.")
```

Comment:
Shall it be an internal config or not? Do we expect this one to be user-facing and tune it frequently?

Reply:
Thanks for reviewing!
I think this should be similar to `PARQUET_FILTER_PUSHDOWN_ENABLED` and be a user-facing config. I guess we can default it to true in the future after we have more testing.